ale_linters/java/javac.vim

```diff
@@ -0,0 +1,106 @@
+" Author: farenjihn <farenjihn@gmail.com>
+" Description: Lints java files using javac
+
+if exists('g:loaded_ale_linters_java_javac')
+    finish
+endif
+
+let g:loaded_ale_linters_java_javac = 1
+let g:ale_java_javac_classpath = ''
+
+let s:eclipse_classpath = ''
+let s:tmppath = '/tmp/java_ale/'
```

Comment:
Yeah, the current one will work. I still want to rewrite it to use the correct filename but `anything.class` should work for now.